src/test/resources/com/puppycrawl/tools/checkstyle/checks/blocks/InputRightCurlyAloneOrSingleline.java

```diff
@@ -105,7 +105,7 @@ void doDoubleBraceInitialization() {
             put("first", "second");
             put("polygene", "lubricants");
             put("alpha", "betical");
-        }}; // it's ok
+        }}; //violation
```

Comment:
I can make it to be violation if needed ! But I felt that it violated the philosophy of `ALONE_OR_SINGLELINE` ! But now that you have explained why is it so in a comment in issue 4057 I understand it. Should I revert this too ?